src/components/LoginForm/PersonalFormFields.tsx

```diff
@@ -0,0 +1,120 @@
+import { generateClient } from "aws-amplify/api";
+import type { AuthUser } from "aws-amplify/auth";
+import { useRouter } from "next/navigation";
+
+import type { Schema } from "@/amplify/data/resource";
+import FormFields from "@/components/LoginForm/FormFields";
+import FormFieldsHeader from "@/components/LoginForm/FormFieldsHeader";
+import { Flex, Input, Label, SelectField } from "@aws-amplify/ui-react";
+import { useMutation, useQuery } from "@tanstack/react-query";
+
+const client = generateClient<Schema>();
+export default function PersonalFormFields({
+  user,
+}: {
+  user: AuthUser | undefined;
+}) {
+  const router = useRouter();
+  const userMutation = useMutation({
+    mutationFn: async (input: Schema["User"]["type"]) => {
+      (await client.models.User.create(input)).data;
+    },
+    onSuccess: () => {
+      // TODO: ADD TOAST
+      router.push("/");
+    },
+  });
+  const institutions = [
+    "University of Calgary",
+    "Mount Royal University",
+    "SAIT",
+    "Other",
+    "None",
+  ];
+  // Not really typesafe, but it's for a hackathon...
+  const submitForm = (formData: FormData) => {
+    const data = Object.fromEntries(formData.entries());
```

Comment:
If we use a useState to manage this form's data, we can have type support for data. Right now data has an ambiguous type which could be dangerous. It would be best that we use the method of form data handling used here `origin/HMT-10/User-Profile` across the whole site.